FizzBuzz.py

```diff
@@ -0,0 +1,49 @@
+#FizzBuzz by M.Horsler 4/10/23. Utilise Lists to allow manipulation and concatenation to provide desired result.
+#Print to Screen function
+def Screen(text):
+    print (text)
+
+#Define variables
+Loop = 1
+result_list=[]
+
+while Loop<256:
```

Comment:
Minor, but this feels like a very natural place for a for-loop since we know when the loop will stop in advance:
```suggestion
for Loop in range(1, 256):
    result_list=[]
```
I'd also set/reset the result_list here, since it's easy to forget resetting things at the end, but creating it inside the loop in the first place will achieve the same thing